components/StatusModal.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div ref="statusDialog" class="text-center">
+        <v-dialog v-model="dialog" width="500">
```

Comment:
For `Boolean` properties, it's a good practice to prefix with a verb. Like `showStatusDialog` or `isStatusDialogOpen`

Reply:
Yes understood, ill do the required changes and commit.